### PR TITLE
binding threshold exposed as parameter

### PR DESCRIPTION
**Pull request by chrisamiller**

### CODE REVIEW

**Comment by tmooney** [comment]
+1 :boom:

File: lib/perl/Genome/Model/Tools/EpitopePrediction/BindingFilter.pm [hunk 1]
```diff
@@ -27,6 +27,11 @@ class Genome::Model::Tools::EpitopePrediction::BindingFilter {
             doc => 'Minimum fold change between mutant binding score and wild-type score. The default is 0, which filters no results, but 1 is often a sensible default (requiring that binding is better to the MT than WT)',
             default => 0,
         },
+        binding_threshold => {
+            is => 'Numeric',
```
Comment (dufeiyu) [inline lib/perl/Genome/Model/Tools/EpitopePrediction/BindingFilter.pm]: Numeric or Number ?

Comment (chrisamiller) [inline lib/perl/Genome/Model/Tools/EpitopePrediction/BindingFilter.pm]: minimum_fold_change, right above it is "Numeric". Is one preferred over the other? Is there a difference?

Comment (dufeiyu) [inline lib/perl/Genome/Model/Tools/EpitopePrediction/BindingFilter.pm]: Both should be Number. There is no UR::Value::Numeric. UR treats that unrecognized data type as SloppyPrimitive.

**Comment by chrisamiller** [comment]
Both sites updated to "Number". Thanks for the catch, Feiyu!

**Comment by dufeiyu** [comment]
+1

**Pull request merged**